frontend/src/lib/components/KanidmSetting.svelte

```diff
@@ -0,0 +1,73 @@
+<script lang="ts">
```

Comment:
This file is mostly copy/pasted from `AutheliaSetting.svelte`, but with a few changes for the URL binding